src/styles/blocks/catalog.scss

```
@@ -4,10 +4,13 @@
   align-items: center;
 
   &__box {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
+    display: grid;
+    justify-content: center;
+    gap: 0 48px;
```

Думаю варто забрати margin-bottom 50px за карточки та вказати тут (gap)

---

use row-gap for grid container instead of margin-top for every card
```suggestion
    gap: 50px 48px;
```